src/App.tsx

```diff
@@ -15,48 +16,140 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  SortByAlphabet = 'alphabet',
+  SortByLength = 'length',
+  DefaultValue = '',
+}
```

Comment:
In the name of the `enam`, you already indicated that it will be sorting, so the names can be made a little shorter:
```suggestion
enum SortType {
  ByAlphabet = 'alphabet',
  ByLength = 'length',
  DefaultValue = '',
}
```